lib/ui/views/transfer/layouts/components/transfer_textfield_message.dart

```diff
@@ -0,0 +1,67 @@
+/// SPDX-License-Identifier: AGPL-3.0-or-later
+part of '../transfer_sheet.dart';
+
+class TransferTextFieldMessage extends ConsumerStatefulWidget {
```

Comment:
J'ai l'impression que ce widget pourrait être stateless.

Reply:
no. we need for the moment to dispose controller